README.md

```diff
@@ -460,10 +460,9 @@ db.createReadStream({ keys: false, values: true })
 <a name="writeStreams"></a>
 #### What happened to `db.createWriteStream`?
 
-Yes we have removed `db.createWriteStream` but not to worry, there is good reason for this. *Disclaimper*: if you are not in a stage where you are worried about performance but want a streaming interface
-into your database, please checkout [`level-ws`][level-ws].
+`db.createWriteStream()` has been removed in favor of a smaller core. The main reason for its existence was sake of symmetry with `db.createReadStream()`. For more information behind this move, please read the following [thread](https://github.com/rvagg/node-levelup/issues/199).
 
-TODO: talk about performance and multiple writeStream implementations
+Also, check out the [modules page](https://github.com/rvagg/node-levelup/wiki/Modules) in the wiki for examples on write stream modules in userland.
```

Comment:
We should add a specific `write-stream`s section to link to directly to make it a bit easier.